dali/pipeline/pipeline_test.cc

```diff
@@ -227,19 +227,28 @@ TYPED_TEST(PipelineTest, TestExternalSource) {
   Pipeline pipe(batch_size, num_thread, 0);
 
   pipe.AddExternalInput("data");
+  pipe.Build({{"data", "cpu"}});
```

Comment:
This file didn't get the license update for some reason?

Reply:
probably because I started working on it before I added a git hook.